src/polyswarm/client/sandbox.py

```diff
@@ -72,15 +72,18 @@ def task_latest(ctx, sandbox_, sha256):
 @sandbox.command('search', short_help='Search for all the SandboxTasks.')
 @click.argument('sha256', type=click.STRING, required=True)
 @click.option('--sandbox', 'sandbox_', type=click.STRING)
+@click.option('--start-date', 'start_date', type=click.STRING)
+@click.option('--end-date', 'end_date', type=click.STRING)
+@click.option('--status', 'status', type=click.STRING)
 @click.pass_context
-def task_list(ctx, sha256, sandbox_):
+def task_list(ctx, sha256, sandbox_, start_date, end_date, status):
     """
     Search for all the SandboxTasks identified by the tuple (hash, community, [sandbox], [start_date], [end_date]).
```

Comment:
Should we update the docstring to also include the status, or should we remove the mention of options, letting the person call --help ?

Reply:
good catch! updated in https://github.com/polyswarm/polyswarm-cli/pull/135/commits/b4aa183738c7234454a6ece216be5374e40f751f